src/main/scala/outwatch/dom/Compat.scala

```diff
@@ -1,12 +1,12 @@
 package outwatch.dom
 
 import cats.effect.IO
+import com.raquo.domtypes.generic.defs.reflectedAttrs
 import org.scalajs.dom.{ClipboardEvent, DragEvent, KeyboardEvent, MouseEvent}
-import outwatch.dom.helpers.InputEvent
+
+import scala.language.higherKinds
```

Comment:
These imports are not necessary, as we have all language features enabled via a [compiler flag](https://github.com/OutWatch/outwatch/blob/master/build.sbt#L35)

Reply:
That's correct. Intellij is was smart enough to realise that :)